docs/index.md

```diff
@@ -139,7 +142,7 @@ configuration by your self:
 * Run `mkcert -install` (you can use the shortcut from the start menu for that)
 * Run `mkcert -CAROOT` to see the local folder used for the newly created root
   certificate authority
-* Open the Firefox Preferences
+* Open Firefox Preferences
```

Comment:
Add `(about:preferences#privacy)`?

Reply:
Yes, for these things it will be easier if you use the "suggest" thing to create a commit.
<img width="824" alt="Cursor_and__docs__Improve_installation_instructions__especially_for_WSL2_Linux_by_uberhacker_·_Pull_Request__3579_·_drud_ddev" src="https://user-images.githubusercontent.com/112444/154817090-30de2b2a-f2a5-4222-a836-0d8b1af2259a.png">

Reply:
```suggestion
* Open Firefox Preferences (about:preferences#privacy)
```